src/constants.py

```diff
@@ -1,5 +1,8 @@
+from src.types import Gwei, SlotNumber
+
 # https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#misc
-FAR_FUTURE_EPOCH = 2 ** 64 - 1
+GENESIS_SLOT = SlotNumber(0)
```

Comment:
Consider using `Slot` name for the type instead of `SlotNumber` to align with the CL spec

Reply:
SlotNumber is more obvious name. Decided to use old version